pages/learn/json-schema-examples.md

```diff
@@ -41,7 +63,126 @@ title: JSON Schema examples
 }
 ```
 
+**Data**
+
+```json
+{
+  "post-office-box": "123",
+  "street-address": "456 Main St",
+  "locality": "Cityville",
+  "region": "State",
+  "postal-code": "12345",
+  "country-name": "Country"
+}
+```
+
+
+## Blog post
+
+A schema representing a blog post, including properties like `title`, `content`, `published date`, `author details`, and `tags`.
```

Comment:
If you're going to use the "speech" form of these properties (e.g. "published date" instead of `publishedDate`), I think it should be without formatting.  Otherwise, just use the actual property names.  Similarly for the other summaries.  We should be consistent as well.

Reply:
How about using the publishedDate form instead? Having a space in a key is not something that I have seen widely used in json in my experience.

Reply:
Yes, that would fall under

> Otherwise, just use the actual property names.